__tests__/app.test.js

```diff
@@ -55,13 +48,25 @@ describe("GET /api/topics", () => {
       });
   });
 });
-describe("GET /api/articles", () => {
+describe.only("GET /api/articles/:article_id", () => {
   test("200: responds with an array of articles", () => {
     return request(app)
-      .get("/api/articles")
+      .get("/api/articles/1")
       .expect(200)
-      .then(({ body: { articles } }) => {
-        expect(articles).toEqual(endpointsJson);
+      .then(({ body }) => {
+        console.log(body);
+        body.rows.forEach((articles) => {
```

Comment:
If this array is length 0, an expect statement is never called, which means if the array is empty then this test always passes. Is there another way you could check it?